R/model.R

```diff
@@ -86,11 +86,11 @@ get_model <- function(workspace,
 register_model <- function(workspace,
                            model_path,
                            model_name,
+                           datasets,
```

Comment:
could you move up the documentation of datasets after model_name?

Reply:
yep!